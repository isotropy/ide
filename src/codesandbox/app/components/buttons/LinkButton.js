import styled from 'styled-components';
import theme from 'codesandbox/common/theme';

export default styled.button`
  display: inline-block;
  background-color: transparent;
  color: ${() => theme.secondary()};
  border: none;
  outline: none;
  cursor: pointer;
  text-decoration: underline;
`;
