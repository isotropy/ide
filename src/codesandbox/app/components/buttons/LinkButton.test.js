import React from 'react';
import testRender from 'codesandbox/app/utils/test/render';
import LinkButton from './LinkButton';

describe('LinkButton', () => {
  it('renders', () => {
    testRender(<LinkButton>Test</LinkButton>);
  });

  it('renders onClick', () => {
    testRender(<LinkButton onClick={() => {}}>Test</LinkButton>);
  });
});
