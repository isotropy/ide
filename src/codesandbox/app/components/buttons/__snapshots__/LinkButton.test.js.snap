// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkButton renders 1`] = `
.c0 {
  display: inline-block;
  background-color: transparent;
  color: #6CAEDD;
  border: none;
  outline: none;
  cursor: pointer;
  text-decoration: underline;
}

<button
  className="c0"
>
  Test
</button>
`;

exports[`LinkButton renders onClick 1`] = `
.c0 {
  display: inline-block;
  background-color: transparent;
  color: #6CAEDD;
  border: none;
  outline: none;
  cursor: pointer;
  text-decoration: underline;
}

<button
  className="c0"
  onClick={[Function]}
>
  Test
</button>
`;
